collector/prom-native-clickhouse/docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: '3.7'
+
+services:
+  otel-collector:
+    container_name: otel-collector
+    image: otel/opentelemetry-collector-contrib:0.56.0
+    command: [ "--config=/conf/collector.yml" ]
+    ports:
+      - 8888:8888
+    environment:
+      LS_ACCESS_TOKEN: ${LS_ACCESS_TOKEN}
+    volumes:
+      - ./collector.yml:/conf/collector.yml:rw
+    depends_on:
+      - clickhouse
+
+  clickhouse:
+    image: yandex/clickhouse-server
+    container_name: clickhouse-server
+    restart: always
+    ports:
+      - 8001:8001
+      - 8123:8123
+      - 9000:9000
+      - 9009:9009
+    volumes:
```

Comment:
See if you can quickly incorporate a healthcheck on this one similar to what we discussed on cockroach. If it isn't panning out in < 30 minutes of effort then ping me. From what I saw in logs this one is less critical, but on the other hand should be able to make short work of it having the recent experience on cockroach.

Reply:
@nslaughter added it.